test/simplifyExpression/distributeSearch/distributeSearch.test.js

```diff
@@ -69,5 +64,21 @@ describe('distribute with fractions', function () {
     ],
   ];
 
-  tests.forEach(t => testDistributeResult(t[0], t[1]));
+  const multiStepTests = [
+
+    ['(2 / x +  3x^2) * (x^3 + 1)',
+      ['(2 * (x^3 + 1) / x + 3x^2 * (x^3 + 1))',
```

Comment:
Oh, good point - I had thought that given `a * b / c`, the `a * b` is considered the numerator. Is that not the case? If not, how would I fix that? Is there a way to explicitly parenthesize an expression? I think on lines 19 and 20 the order of ops is already correct...?